app/model/ReminderNotice.scala

```diff
@@ -33,6 +34,8 @@ sealed case class ReminderNotice(hoursBeforeReminder: Int, progressStatuses: Pro
     case PHASE2_TESTS_FIRST_REMINDER => (DAYS, Phase_2)
     case PHASE3_TESTS_SECOND_REMINDER => (HOURS, Phase_3)
     case PHASE3_TESTS_FIRST_REMINDER => (DAYS, Phase_3)
+    case status => throw UnrecognisedReminderNoticeProgressStatusException(s"$status is an unrecognised progress status " +
+      s"for reminder notices")
   }
 
   val timeUnit: TimeUnit = timeUnitAndPhase._1
```

Comment:
I know this wasn't you, but this line and the next is bugging me. Any chance you can change it to:
'val (timeUnit, phase) = timeUnitAndPhase'? Otherwise I can do it and you can code review me(?)